resources/dev/scripts/telemetry.sh

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env bash
+CUR_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" &>/dev/null && pwd)"
+source "${CUR_DIR}/env.sh"
+
+# Run the below command to retrieve the latest version
+# curl -s "https://api.github.com/repos/prometheus-operator/prometheus-operator/releases/latest" | jq -cr .tag_name
+
+function fetch-prometheus-operator-bundle() {
```

Comment:
I believe prometheus operator can be installed using prometheusStack in the fullstack-cluster-setup helm chart, what is the difference here?

Reply:
Well, I could not get the metrics exported to prometheusStack in fullstack-cluster-setup. These scripts just gives us an working example where we can see some metrics. We need to work on a better solution for metrics export using grafana-agent (which I have just started figuring out). As I said earlier, I'll take out the dev folder and put it somewhere else.